jib-core/src/main/java/com/google/cloud/tools/jib/frontend/ExposedPortsParser.java

```diff
@@ -70,7 +69,7 @@
       if (!Strings.isNullOrEmpty(matcher.group(2))) {
         max = Integer.parseInt(matcher.group(2));
       }
-      String protocol = matcher.group(3);
+      Protocol protocol = "udp".equals(matcher.group(3)) ? Protocol.UDP : Protocol.TCP;
```

Comment:
can we compare directly with the enum for `Protocol.UDP` instead of `"udp"` here?

Reply:
So like `Protocol.UDP.toString().equals(...)`?

Reply:
I think so? You might need to play around with it a little.